.github/workflows/style.yml

```diff
@@ -1,28 +1,31 @@
+---
 name: Style
 
 on:
   push:
     branches:
       - master
+      - develop
```

Comment:
We don't work with a develop branch, but rather feature branches and PRs. (same below)

```suggestion
```

Reply:
I am working with git  flow.  So I develop on develop.

Master is yours ...